vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -581,14 +581,17 @@ public void emit(SectionRenderer sectionRenderer)
                     e ->
                         new Target() {
                           private final List<Pair<String, ConsumableResource>> consumables =
-                              Stream.concat(
+                              Stream.concat(Stream.concat(
                                       e.getValue().getConsumableResources().stream()
                                           .map(
                                               name ->
                                                   new Pair<>(name, consumableResources.get(name))),
                                       Stream.of(
                                           new Pair<String, ConsumableResource>(
-                                              "", maxInFlightPerWorkflow)))
+                                              "", maxInFlightPerWorkflow))),
+                                    Stream.of(
```

Comment:
Stream.of takes variable values (https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/stream/Stream.html#of(T...)), have 1 Stream.concat and expand the existing Stream.of to include both new Pairs